src/models/apps/ecommerce/order.models.js

```diff
@@ -48,8 +48,7 @@ const orderSchema = new Schema(
       default: [],
     },
     address: {
-      type: Schema.Types.ObjectId,
-      ref: "Address",
+      type: Object,
```

Comment:
Can we add more safe type instead of `Object`? Like specific fields we will be having in the address field?

Reply:
Sure, I will update it.

But i also think there is no need for it, because, we are already having validations and safety added on address modal, and we are directly using that for address selection.

Reply:
Hi @wajeshubham I will update it, 

But also I dont think it will be necessary as we are directly using Address Modal which has safe types and validations on it.

Reply:
@Sarthak-ONS @wajeshubham the specified field in the address field would be okay...I don't know if you went through the pull request i sent in earlier that was closed, I literally explained it well there..